src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -15,13 +15,7 @@ namespace Microsoft.DotNet.Cli
 {
     internal static class TestCommandParser
     {
-        public static readonly string DocsLink = "https://aka.ms/dotnet-test";
-
-        public static readonly Argument<string> SlnOrProjectArgument = new Argument<string>(CommonLocalizableStrings.SolutionOrProjectArgumentName)
-        {
-            Description = CommonLocalizableStrings.SolutionOrProjectArgumentDescription,
-            Arity = ArgumentArity.ZeroOrOne
-        };
+        public static readonly string DocsLink = "https://aka.ms/dotnet-test";        
```

Comment:
Nit: Looks like I somehow added extra trailing spaces.

```suggestion
        public static readonly string DocsLink = "https://aka.ms/dotnet-test";
```